projects/01bsp_imu/01bsp_imu.c

```diff
@@ -0,0 +1,52 @@
+/**
+ * @file 01bsp_i2c.c
+ * @author Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @brief This is a short example of how to use the I2C api.
+ *
+ * @copyright Inria, 2022
+ *
+ */
+#include <nrf.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "board.h"
+#include "timer_hf.h"
+#include "ism330.h"
+
+//=========================== defines ==========================================
+
+
+
+//=========================== variables ========================================
+
+ism330_acc_data_t acc_data;
+ism330_gyro_data_t gyro_data;
```

Comment:
These variables should be made static (only visible in the application scope):
```suggestion
static ism330_acc_data_t acc_data;
static ism330_gyro_data_t gyro_data;
```